test/basic/init_funcs.c

```diff
@@ -0,0 +1,21 @@
+#include "smack.h"
+
+// @expect verified
+// @flag --unroll=1
+
+void __SMACK_INIT(defineProcessStates) {
+  __SMACK_top_decl("const unique $process_uninitialized: int;");
```

Comment:
I feel that this is slightly...  misleading?  I mean, these are all things you can do without __SMACK_INIT...

I guess I was trying to find something that couldn't be done without __SMACK_INIT...  But then again, I guess the whole purpose of it is to use from .h files, not from actual end-user files.